CMakeLists.txt

```diff
@@ -87,6 +87,13 @@ add_executable(moonlight ${SRC_LIST})
 target_link_libraries(moonlight m)
 target_link_libraries(moonlight gamestream)
 
+if (CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
+  set(ALSA_FOUND FALSE)
+  set(PULSE_FOUND FALSE)
+  set(CEC_FOUND FALSE)
```

Comment:
Is it correct to force `CEC_FOUND` to `FALSE` here? You can have libcec on FreeBSD, right?

Reply:
Yes, FreeBSD has libcec.I just found out that I can control options with CMAKE_ARGS, and I'll remove it from the code.